v20.1/time.md

```diff
@@ -63,6 +63,16 @@ A `TIME` column supports values up to 8 bytes in width, but the total storage si
 
 A `TIMETZ` column supports values up to 12 bytes in width, but the total storage size is likely to be larger due to CockroachDB metadata.
 
+## Precision
```

Comment:
wondering if we should do examples for this in the below section (e.g. something like https://sourcegraph.com/github.com/cockroachdb/cockroach@793d487107c6a8f61070ef0027e6d0574edb9e07/-/blob/pkg/sql/logictest/testdata/logic_test/timestamp#L85). otherwise, LGTM!